app/.prettierrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  singleQuote: true,
+  trailingComma: 'none',
```

Comment:
I tend to be pretty easygoing with formatting rules, but I would like to know what is so terrible about trailing commas? I personally appreciate how they minimize git diffs by not necessitating that the previous line be edited when adding a new properties to objects.

Reply:
I have no issue with one or the other, just wanted to avoid changing every single file in this PR. I intend to remove this file in the next PR, and let prettier do its thing